### PR TITLE
updated boilerplate with mongodb

### LINKED ISSUE #9: add mongodb in boilerplate

### PR DESCRIPTION
**Pull request by aaa006bd**
Closes #9

### CODE REVIEW

File: app/backend/package.json [hunk 1]
```diff
@@ -24,13 +24,18 @@
...
     "cors": "^2.8.5",
     "debug": "^4.1.1",
+    "feathers-mongoose": "^8.3.1",
     "helmet": "^3.21.2",
+    "mongoose": "^5.10.2",
     "mysql": "^2.18.1",
     "pg": "^8.0.3",
     "serve-favicon": "^2.5.0",
+    "typegoose": "^5.9.1",
     "typeorm": "^0.2.22",
```
Comment (lukashass) [inline app/backend/package.json]: Should the `typeorm` dependency be deleted?

File: app/backend/src/database.ts [hunk 2]
```diff
@@ -1,53 +1,45 @@
...
-    migrationsRun: true, // run migrations on app start
-    migrations: [],
-    migrationsTransactionMode: 'each',
-
-    // misc
-    synchronize: true, // this should synchronize all definded entities with the db, disables this when using migrations
-    logging: false,
-  };
 
   // create a typeorm connection
```
Comment (lukashass) [inline app/backend/src/database.ts]: Comment still mentions typeorm

File: app/backend/src/database.ts [hunk 3]
```diff
@@ -1,53 +1,45 @@
...
 
-    await new Promise((resolve) => setTimeout(resolve, RECONNECT_TIMEOUT));
+//     await new Promise((resolve) => setTimeout(resolve, RECONNECT_TIMEOUT));
 
-    return openDatabase(connectionOptions);
-  }
-};
+//     return openDatabase(connectionOptions);
+//   }
+// };
```
Comment (lukashass) [inline app/backend/src/database.ts]: These commented lines could be removed

**Review by lukashass — CHANGES_REQUESTED** [review]
Things to solve before merging:

- [x] Remove occurrences of `typeorm`
- [x] fix `yarn lint`
- [x] fix `yarn test:unit`

File: app/backend/package.json (hunk 1 above)
Comment (aaa006bd) [reply app/backend/package.json]: I think so.

File: app/backend/package.json [hunk 4]
```diff
@@ -36,7 +36,6 @@
     "pg": "^8.0.3",
     "serve-favicon": "^2.5.0",
     "typegoose": "^5.9.1",
-    "typeorm": "^0.2.22",
```
Comment (lukashass) [inline app/backend/package.json:34]: @aaa006bd When you remove this you also have to update the lock file by running `yarn install`

**Review by lukashass — CHANGES_REQUESTED** [review]
You didn't solve these, yet.
- [x] Remove occurrences of `typeorm`
- [x] fix `yarn lint`
- [x] fix `yarn test:unit`

**Review by lukashass — APPROVED** [review]

**Pull request merged**